exercises/wordy/example.rs

```diff
@@ -18,74 +12,57 @@ impl Token {
     }
 
     fn is_operator(&self) -> bool {
-        self.value == String::from("plus")
-            || self.value == String::from("minus")
-            || self.value == String::from("multiplied")
-            || self.value == String::from("divided")
+        self.value == "plus"
+            || self.value == "minus"
+            || self.value == "multiplied"
+            || self.value == "divided"
     }
 }
 
-pub fn answer(c: &str) -> Option<isize> {
-    WordProblem::new(c).answer()
-}
+pub fn answer(c: &str) -> Option<i32> {
+    let mut t = tokens(c);
+    let mut result: i32 = 0;
+    let mut opr = "plus";
 
-impl WordProblem {
-    fn new(c: &str) -> Self {
-        WordProblem {
-            command: String::from(c),
+    if t.len() <= 1 {
+        None
```

Comment:
I'm surprised at this behavior. Not saying you're wrong: this keeps compatibility with the previous implementation. All the same, I'd expect `answer("42") == Some(42)`. Do you think we should change this behavior and add a test case?

Reply:
Although I would consider the string "42" to not be a fully-formed word problem, one could imagine "What is 42?" to be a valid problem and have an answer of 42. It depends on whether a valid word problem is required to have at least one operation in it.

My default mode of operation is not to be too interested and therefore to be content with leaving "What is 42?" as undefined behaviour, but at this current moment I am also reasonably able to start a discussion in problem-specifications and see where it goes.